### PR TITLE
Export transition component

### PR DESCRIPTION
**Pull request by toha993**

### CODE REVIEW

File: packages/react/src/index.ts [hunk 1]
```diff
@@ -19,3 +18,4 @@ export * from "./styles";
 export * from "./text";
 export * from "./tokens";
 export * from "./tooltip";
+export * from "./transition";
```
Comment (MunifTanjim) [inline packages/react/src/index.ts:21]: Are we sure we want to export it with the current interface? Once it's published it'd be hard to change it.

Comment (toha993) [reply packages/react/src/index.ts:21]: Got it vai. Should I Close the PR?

Comment (MunifTanjim) [reply packages/react/src/index.ts:21]: cc: @radnan

Comment (radnan) [reply packages/react/src/index.ts:21]: we'll need to document it - but i think the API is good for now - so we can merge this

Comment (radnan) [reply packages/react/src/index.ts:21]: we should also export everything from `animate-presence` since those two go together

Comment (toha993) [reply packages/react/src/index.ts:22]: I think we need to change something in web components, it's getting failed in the build.
C.C @radnan

**Review by radnan — APPROVED** [review]

**Pull request merged**